packages/react/ds-core/.storybook/preview.ts

```diff
@@ -6,6 +6,7 @@ import "@canonical/styles-debug/baseline-grid";
 
 const preview: Preview = {
   decorators: [
+    // @ts-ignore. Unsure why this type error is happening only on this package. Must cleanup when upgrading to Storybook 9.
```

Comment:
Adding a reference to the [storybook upgrade pr](https://github.com/canonical/pragma/pull/242) for visibility on that PR, so we know to fix this then.